src/components/OnHandleErrorClick.js

```diff
@@ -5,11 +5,13 @@ const OnHandleErrorClick = (targetName) => {
       document.getElementById(targetName) ||
       document.getElementById("establishment_opening_date");
     targetTradingDate.scrollIntoView();
+    targetTradingDate.focus({ preventScroll: true });
```

Comment:
Is there a reason for using scrollIntoView and then preventScroll on the focus?

Reply:
just to ensure when it's invoked it doesn't scroll again (although it'd probably just stay in the same place) as some browsers invoke a scroll automatically with the focus. 
Tbh it's not that necessary so could remove it and it'd likely not make a difference, just read it somewhere that it makes it smoother